### PR TITLE
1327 binarizemojo test

### PR DESCRIPTION
**Pull request by levBagryansky**
This PR related to #1327 
I extented `FakeMaven` to test `BinarizeMojo` in future and use it in simple test `BinarizeMojoTest`.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java [hunk 1]
```diff
@@ -50,6 +51,11 @@
 @SuppressWarnings("PMD.LongVariable")
 public final class BinarizeMojo extends SafeMojo {
 
+    /**
+     * The directory where to binarize to.
+     */
+    public static final String DIR = "binarize/";
```
[the remark this answers is not in the document]
Comment (levBagryansky) [reply eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java:57]: @Graur `binarize` and `binarize/` mean the same directory. Is it incorrect?

**Review by Graur — APPROVED** [review]
@levBagryansky Looks good to me, thanks!
@yegor256 Please, have a look

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@levBagryansky this is not a fully working implementation, is it? I guess, you should add a puzzle to your code, so that a ticket will be created when we merge the PR

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**